src/dataTransfer/index.ts

```diff
@@ -61,68 +64,95 @@ export class DataTransferManager {
 		}
 	}
 
-	public handleCommand(command: Commands.IDeserializedCommand): void {
-		const allLocks = [this.stillsLock, ...this.clipLocks]
-
-		// try to establish the associated DataLock:
-		let lock: DataLock | undefined
-		if (
-			command.constructor.name === Commands.LockObtainedCommand.name ||
-			command.constructor.name === Commands.LockStateUpdateCommand.name
-		) {
-			lock = allLocks[command.properties.index]
-		} else if (typeof command.properties.storeId === 'number') {
-			lock = allLocks[command.properties.storeId]
-		} else if (command.properties.transferId !== undefined || command.properties.transferIndex !== undefined) {
-			for (const _lock of allLocks) {
+	public async handleCommand(command: Commands.IDeserializedCommand): Promise<void> {
```

Comment:
This method doesnt need to be async anymore either.
I think it would be good to rename to queueCommand, to make it clear to callers that it queues things instead of processing immediately